Cargo.toml

```
@@ -53,9 +53,9 @@ serde_json = "1.0"
 serde_yaml = "0.9"
 sha2 = "0.10"
 snafu = { version = "0.8", features = ["futures"] }
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.67.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.67.1" }
 tera = "1.18"
-termion = "3.0.0"
+termion = "4.0.0"
```

Apparently we should be using major.minor...

```suggestion
termion = "4.0"
```

---

I assume it was done. Mobile view requires more effort to see that.

---

485611ae77c22e9c6e3ddcdc18891c067c3a5a58